src/clojure_for_js_devs/redis.clj

```diff
@@ -1,5 +1,6 @@
 (ns clojure-for-js-devs.redis
   (:gen-class)
+  (:refer-clojure :exclude [get])
```

Comment:
Is the :refer-clojure necessary here? I'm surprised that there would be a namespace collision since the `get` in `car/get` is already attached to a namespace.